src/pretalx/orga/templates/orga/auth/base.html

```diff
@@ -23,9 +23,7 @@
     <img src="{% static "common/img/logo_white.svg" %}" class="logo" alt="{% trans "The pretalx logo" %}">
     {% if messages %}
         {% for message in messages %}
-            <div class="card card-inverse card-{{ message.tags }} mb-3 text-center">
-                {{ message }}
-            </div>
+            <div class="card card-inverse card-{{ message.tags }} mb-3 text-center">{{ message }}</div>
```

Comment:
What's the reason for this change?

Reply:
only the debug message -
the debug message was set via css to use `whitespace: pre-line`
to allow for multi-line formatted messages...